index.html

```diff
@@ -1,16 +1,59 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+<title>Responsive Cake webpage</title>
+<link rel="stylesheet" href="./style.css"></head>
 </head>
-
 <body>
-    <!-- Add your markup here -->
+<!-- Add your markup here -->
+<header>
+<section class= "image-container" >
+<div class= "logo">
+    <img src="image/logo.jpg" alt="">
+</div>
+<div class= "textMenu-container" >
+    <img class="menu-icon" src="image/menu-hamburger.svg" alt=""/>
+    <section class="text">
+    <p>Best cake in town </p>
+    <p>Deliver to your door</p>
+</section>
+</div>
+</section>
+ <div class= "hamburger-menu">
+    <div><a class="hamburger-item" href="#Aboutus"><p>About us</p></a></div>
+    <div><a class="hamburger-item" href="#CustomerCare"><p>Customer Care</p></a></div>
+    <div><a class="hamburger-item" href="#TermOfUse"><p>Term of use</p></a></div>
+</div>
+</header>
+<hr>
+<main>
+    <section id="intro-container"> 
+        <div>
+            <h1 class="intro-heading">Welcome</h1>
+            <p class="intr">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque et congue nulla, eu maximus felis. Aenean scelerisque viverra urna, et commodo purus dignissim in. Ut ac interdum risus. Ut interdum, quam et facilisis tincidunt, ex purus molestie libero, at blandit mi ipsum nec elit. Integer sodales ipsum ligula, a venenatis lectus dignissim feugiat. Proin vitae accumsan augue. Ut blandit placerat augue, sit amet lobortis magna. Ut pulvinar elementum purus, sit amet auctor felis mattis in. Sed consectetur, eros at luctus pulvinar, nulla arcu facilisis erat, eu sollicitudin tortor ipsum quis ligula. Suspendisse eu volutpat nisi.</p>
+        </div>  
+        <div>
+            <img class="intro-image" src="image/cake.jpg" alt="Cake picture">
+            
+        </div>
+    </section>
+  
+    <section id="product-container">
+        <div>
+            <div class="product-text"><p>Products</p></div>
+             <a href="#pastries"><img class= "product-image" src="image/pastries.jpg" alt="pastries"></a>
+             <a href="#macaroons"> <img class= "product-image" src="image/macaroon.jpg" alt="macaroons"></a>
+             <a href="#birthdayCakes"><img class= "product-image" src="image/BirthdayCake.jpg" alt="birthday cake"></a>
+             <a href="#cupcakes"><img class= "product-image" src="image/cupcake.jpg" alt="cupcake"></a>
+             <hr>
+        </div>
+    </section>
+</main>
 </body>
-
+<footer>
+    <div id="footer-container">
```

Comment:
Do you need this div? It looks like the `footer` tag could fill this role as it wraps the same content as the div. Move your styles to the `footer` tag (and don't use IDs in your CSS!)